resources/views/Reviews/edit.blade.php

```diff
@@ -0,0 +1,204 @@
+                        <p>店舗の名前</p>
+                    </a>
+                    <button class="detail"><a href="">詳細</a></button>
+                </div>
+                <div class="detail-link">
+                    <a href=""><img src="/css/noimage.png" alt="店舗詳細リンク"></a>
+                    <a href="">
+                        <p>店舗の名前</p>
+                    </a>
+                    <button class="detail"><a href="">詳細</a></button>
+                </div>
+                <div class="spacer"></div>
+                <div class="detail-link">
+                    <a href=""><img src="/css/noimage.png" alt="店舗詳細リンク"></a>
+                    <a href="">
+                        <p>店舗の名前</p>
+                    </a>
+                    <button class="detail"><a href="">詳細</a></button>
+                </div>
+                <div class="detail-link">
+                    <a href=""><img src="/css/noimage.png" alt="店舗詳細リンク"></a>
+                    <a href="">
+                        <p>店舗の名前</p>
+                    </a>
+                    <button class="detail"><a href="">詳細</a></button>
+                </div>
+            </div>
+        </div>
+        {{-- 口コミ編集フォーム --}}
+        <div class="review-form">
+            <form action="{{route('reviews.update',$review->id)}}" method="post">
+                @csrf
+                @method('put')
+                <div class="rate">
+                    <p>評価(★)</p>
+                    <select name="review">
+                        <option value="0" @if($review->rating ==0)selected @endif>☆☆☆☆☆</option>
+                        <option value="1" @if($review->rating ==1)selected @endif>★☆☆☆☆</option>
+                        <option value="2" @if($review->rating ==2)selected @endif>★★☆☆☆</option>
+                        <option value="3" @if($review->rating ==3)selected @endif>★★★☆☆</option>
+                        <option value="4" @if($review->rating ==4)selected @endif>★★★★☆</option>
+                        <option value="5" @if($review->rating ==5)selected @endif>★★★★★</option>
+                    </select>
+                </div>
+                <div class="form-group">
+                    <label>口コミ内容記入</label>
+                    <textarea type="text" class="form-control" id="" cols="110" rows="3" name="comment">{{$review->comment}}</textarea>
+                </div>
+                <div class="form-group">
+                    <label class="space">食べたもの</label>
+                    <input type="text" class="form-control" value="{{$review->ate_thing}}" name="ate_thing">
+                </div>
+                <div class="form-group">
+                    <label class="space">支払額</label>
+                    <input type="text" class="form-control" value="{{$review->charge}}" name="charge">
+                </div>
+                <div class="form-group">
+                    <label class="space">カテゴリ</label>
+                    <select id="sweets" name="categories">
```

Comment:
[MUST]
nameはcategory_idですね
```suggestion
                    <select id="sweets" name="category_id">
```